src/main/java/seedu/address/model/person/Person.java

```diff
@@ -105,14 +102,8 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Phone: ")
-                .append(getPhone())
-                .append(" Email: ")
-                .append(getEmail())
-                .append(" Address: ")
-                .append(getAddress())
-                .append(" Tags: ");
+        builder.append(getName()).append(" Phone: ").append(getPhone()).append(" Email: ").append(getEmail())
+                .append(" Address: ").append(getAddress()).append(" Tags: ");
```

Comment:
Similarly, I believe the original is more reader friendly

Reply:
This complies with the Java Coding Standard.